src/server/debug/server/server.go

```diff
@@ -575,6 +583,20 @@ func (s *debugServer) collectLogs(tw *tar.Writer, pod, container string, prefix
 	}, prefix...)
 }
 
+func (s *debugServer) collectLogsLoki(tw *tar.Writer, pod, container string, prefix ...string) error {
+	return collectDebugFile(tw, "logs-loki", "txt", func(w io.Writer) error {
+		queryStr := `{pod="` + pod
```

Comment:
I was worried for a bit that we might have to worry about special characters but `=` does seem to just be strings and not regex